docs/source/release/v5.0.0/mantidworkbench.rst

```diff
@@ -114,5 +114,6 @@ Bugfixes
 - Fixed bug that caused an error if a MDHistoWorkspace was plotted and a user attempted to open a context menu.
 - Fixed a bug which caused graphic scaling issues when the double-click menu was used to set an axis as log-scaled.
 - Fixed a bug where the colorbar in the instrument view would sometimes have no markers if the scale was set to SymmetricLog10.
+- Fixed a bug which caused the script progress arrow to always start on line 0, regardless of the users selected code.
```

Comment:
This was actually a regression in this dev cycle so we shouldn't need the release note.

Reply:
Okay I've removed it